packages/scaffold-config/src/frameworks.ts

```diff
@@ -61,6 +61,22 @@ function getBundlerDependency (bundler: WizardBundler['type'], projectPath: stri
   }
 }
 
+export const WIZARD_MOUNT_MODULES = ['cypress/react', 'cypress/react18', 'cypress/vue', 'cypress/vue2', 'cypress/angular'] as const
+
+export type WizardMountModule = typeof WIZARD_MOUNT_MODULES[number]
```

Comment:
Any reason we need this in array format? Or could we simplify to something like this?
```
export type WizardMountModule = 'cypress/react' | 'cypress/react18' | 'cypress/vue' | 'cypress/vue2' | 'cypress/angular'
```

Reply:
I believe I was planning to use the array in the test but ended up scrapping it. Good catch, addressed in [143a9d1](https://github.com/cypress-io/cypress/pull/23251/commits/143a9d171d0389a749eac7907842e2d59390a6cf)